exporter/uptraceexporter/exporter_test.go

```diff
@@ -35,17 +36,21 @@ import (
 
 func TestNewTracesExporterEmptyConfig(t *testing.T) {
 	cfg := createDefaultConfig().(*Config)
-	exp, err := newTracesExporter(cfg, zap.NewNop())
+	exp := newTracesExporter(cfg, zap.NewNop())
+	require.NotNil(t, exp)
+
+	err := exp.Start(context.Background(), componenttest.NewNopHost())
 	require.Error(t, err)
-	require.Nil(t, exp)
 }
 
 func TestNewTracesExporterEndpoint(t *testing.T) {
```

Comment:
Not clear the testing case compare with previous test. Probably this needs to be named invalid endpoint and previous test NoEndpoint?